bundles/http/Citrix/Zoom-Meetings/e40510b3-a760-43b4-ab16-56572dc6303b.js

```diff
@@ -3,20 +3,24 @@ const moment = library.load('moment-timezone');
 const uuid = library.load('uuid');
 
 // loading the users data(Done)
-async function syncUsers(dataStore, client, context) {
+async function syncUsers(dataStore, client) {
     const page_size = 100
     let next_page_token = ""
     let userids = [];
     do {
-        const response = await client.fetch(`/users?next_page_token=${next_page_token}&page_size=${page_size}`);
+        const response = await client.fetch(`users?next_page_token=${next_page_token}&page_size=${page_size}`);
         let json = await response.json();
         if (!response.ok) {
-            throw new Error(json);
+            if(json.code === 4700){
```

Comment:
In Jira ticket is "to handle the error code 404".
So why is here some magic constant from json body?

```suggestion
            if (response.status === 404) {
```

Reply:
If the required scopes for the api call are not added in the zoom oauth application, then the sync should not fail. To implement this we have used  error code `4700` which is specific  to scope issues.

Reply:
@ondrejkralik , could you please share the ticket number for "handing the error code 404"

Reply:
MICROAPP-15359
I'm sorry. I overlook that it should be for the Meeting details endpoint -> "We have modified the script to handle the error code 404 for MeetingDetails endpoint"

But then I have a new question: On line 91 you still throw error and do not check status 404. Is it ok?

Reply:
Yes Correct, we need to add the check for that. I missed it somehow.